Sources/Network/Parse.swift

```diff
@@ -55,4 +56,14 @@ public enum Parse {
 
         return image
     }
+
+    /// Parses empty data
+    ///
+    /// - Parameter data: Raw data
+    /// - Throws: A Parse.Error that can be of type
+    public static func void(data: Data) throws {
+        guard data.isEmpty else {
+            throw Error.unexpectedData
```

Comment:
not sure If we need this error, or we can simply use an `assert(data.isEmpty)` 🤔

Reply:
I though it would be better to return an error like the other parsers. It shouldn't happen but as an API I think this is more correct than an assert.